package/deb-pi4-arm64/DEBIAN/control

```diff
@@ -3,5 +3,7 @@ Version: 0.0.1
 Section: utils
 Priority: optional
 Architecture: arm64
-Description: SIMBIoTA Client Daemon
+Description: Simbiota is a lightweight (both memory and CPU usage) antivirus.
 Maintainer: Ukatemi Technologies Zrt.
+Homepage: https://github.com/simbiota/simbiota
+Depends: jq, curl
```

Comment:
If we have dependencies, installing the .deb package would be easies with apt or apt-get instead of dpkg for automatic dependecy installation